src/finder/frameworkSpecs.ts

```diff
@@ -0,0 +1,109 @@
+import { FrameworkSpec } from "./types";
+
+export const frameworkSpecs: FrameworkSpec[] = [
+  {
+    id: "core:express",
+    runtime: "nodejs",
+    webFrameworkId: "Express.js",
+    requiredDependencies: [
+      {
+        name: "express",
+      },
+    ],
+  },
+  {
+    id: "angular",
+    runtime: "nodejs",
+    webFrameworkId: "Angular",
+    requiredFiles: ["next.config.js", "next.config.ts"],
+    requiredDependencies: [
+      {
+        name: "@angular/core",
+      },
+      {
+        name: "@angular/cli",
+      },
+    ],
+    commands: {
+      build: {
+        cmd: "ng build",
+      },
+      dev: {
+        cmd: "ng run",
+        env: { NODE_ENV: "dev" },
+      },
+      run: {
+        cmd: "ng run",
+        env: { NODE_ENV: "production" },
+      },
+    },
+  },
+  {
+    id: "core:nextjs",
+    runtime: "nodejs",
+    requiredFiles: [["next.config.js", "next.config.ts"]],
```

Comment:
next.config is optional

Reply:
Should check for  'pages/' folder be added. As next.js app have at lease one route in your application?